### ISSUE TITLE
MouseSystem on Android doesn't detect clicks sometimes

### ISSUE THREAD

**Issue by Noofbiz** [EngoEngine/engo#567]
The MouseSystem on Android's `engo.Input.Mouse.Action` is for some reason returning `engo.Neutral` rather than `engo.Press`. I managed to work around this by creating my own system that used that, and it worked flawlessly. It only isn't working inside the MouseSystem's Update.

**Comment by Z2Y** [comment]
I  meet the same problem , Any Update ?  
The press and release event fired in same frame.  Action state may lose during update

**Comment by Noofbiz** [comment]
Yeah it should actually be a quick fix, this issue has just gotten a little lost in the shuffle lol. I'll look into this. Thanks!
